frontend/components/Navbar/navbar.module.css

```diff
@@ -0,0 +1,63 @@
+.background {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  background-color: #16202f;
+  align-items: center;
+  justify-content: space-between;
+  height: 9vh;
```

Comment:
This `height` property doesn't do so well on larger and smaller windows:

Larger window (height is too large):
![image](https://user-images.githubusercontent.com/52584399/129259583-6c3e48b3-7d51-4d77-88b7-96121b3010cc.png)


Smaller window (height is too small):
![image](https://user-images.githubusercontent.com/52584399/129259621-a3d5b5e9-6a6c-411b-bdc0-23577fdeaf93.png)

I'd recommend setting it to a static value, such as `72px`.
```suggestion
  height: 72px;
```